packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-polygon-settings.tsx

```diff
@@ -52,13 +53,35 @@ const LockedPolygonSettings = (props: Props) => {
         fillStyle,
         strokeStyle,
         labels,
+        ariaLabel,
         expanded,
         onToggle,
         onChangeProps,
         onMove,
         onRemove,
     } = props;
 
+    function getPrepopulatedAriaLabel() {
+        let str = `Polygon with ${points.length} sides, vertices at `;
+
+        // Add the coordinates of each point to the aria label
+        str += points.map(([x, y], _) => `(${x}, ${y})`).join(", ");
```

Comment:
```suggestion
        str += points.map(([x, y]) => `(${x}, ${y})`).join(", ");
```

Is this underscore argument needed?